core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tracker/UserStoreManagerRegistry.java

```diff
@@ -111,4 +112,39 @@ public static Properties getUserStoreProperties(String className) {
         properties = getUserStoreManagers().get(className);
         return properties;
     }
+
+    /**
+     * Get if the user store is local or not.
+     *
+     * @param className User store name.
+     * @return boolean true if user store is a local.
+     */
+    public static boolean isLocalUserStore(String className) throws UserStoreException {
+
+        Object[] userStoreManagers = userStoreManagerTracker.getServices();
+        for (Object userStoreManagerObj : userStoreManagers) {
+            UserStoreManager userStoreManager = (UserStoreManager) userStoreManagerObj;
+            if (userStoreManager.getClass().getName().equals(className)) {
```

Comment:
Let's use `StringUtils` to avoid possible NPE here.

Reply:
fixed bd3618838bc1db216d4459f5514ad7b03f0792d6